src/components/forms/inputs/TextInput.jsx

```diff
@@ -5,16 +5,23 @@ import InputBase from './InputBase';
 
 @injectIntl
 export default class TextInput extends InputBase {
+
+    constructor(props) {
+    super(props);
+    }
```

Comment:
This is a remnant from your earliest attempt I believe?

Reply:
I tried adding a state prop, but the constructor and the rest was already there as far as I remember. Should I remove it anyway?

Reply:
Thanks for removing it!

The constructor was added in your commit b349b3d1c3c56d5651f5bcde45236b63455b4e43. You can click to see the diff.  It's so easy to forget what was done between two commits, especially when doing trial-and-error of different approaches.

I recommend always using `git add -p` to iterate over all changes and explicitly decide whether they should be part of your commit or not.

It's at least a good idea to inspect what you're about to commit using `git diff` and `git diff --cached`, to make sure that you're not committing any changes that are not required for the work that you're doing.

Maybe your editor (VS Code?) has other good ways of indicating this. I just like to be as close as possible to the code, which is why I like the `git` command-line interface. 😄

Reply:
Thanks! I'll do that from now on. I just got very used to `git add .` even if I know there are other ways. Sounds good being more aware of which changes are being commited.

Reply:
I would very strongly recommend against ever using `git add .`. 😄 

When you do, it's very easy to accidentally add files that shouldn't be tracked at all, e.g. swap files or other temporary files, any notes or other personal files you may have put in the working directory, and so on.

On option is `git add -N .`, which will tell Git to care about all files, but not stage them for commit. After that, you can do `git add -p` to go over all new files (and other changes) one by one.

Reply:
That is good to know! Thanks!